### PR TITLE
linux build instructions

### PR DESCRIPTION
**Pull request by CedricGuillemet**
I may have missed something or a troubleshooting info.
Please try on your side and report me any issue/missleading info.
I was able to run on device and emulator so if something goes wrong, it's only a matter of path or missing package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,32 @@ export JAVA_HOME=$(/usr/libexec/java_home -v 13)
...
+
+**Required Tools:** [Android Studio](https://developer.android.com/studio/) (including NDK 21.3.6528147)
+
+With Ubuntu, you can install needed packages by this command:
+
+```
+sudo apt-get install adb ninja-build openjdk-14-jdk android-sdk
+```
+
+Update PATH with this commands:
```
Comment (ryantrem) [inline README.md:100]: Can this be put into some file like on MacOS so it doesn't have to be done every time you open a terminal?

File: README.md [hunk 2]
```diff
@@ -87,6 +87,32 @@ export JAVA_HOME=$(/usr/libexec/java_home -v 13)
...
+```
+export ANDROID_HOME=$HOME/Android/Sdk
+export PATH=$PATH:$ANDROID_HOME/emulator
+export PATH=$PATH:$ANDROID_HOME/tools
+export PATH=$PATH:$ANDROID_HOME/tools/bin
+export PATH=$PATH:$ANDROID_HOME/platform-tools
+```
+
+**Troubleshootings:**
+- If the metro server is not started with `npm run android` , you can start it manually by running `npm run start` in a terminal.
```
Comment (ryantrem) [inline README.md:111]:
```suggestion
- If the Metro server is not started with `npm run android` , you can start it manually by running `npm run start` in a terminal.
```

File: README.md [hunk 3]
```diff
@@ -87,6 +87,32 @@ export JAVA_HOME=$(/usr/libexec/java_home -v 13)
...
+- Android studio is the tool of choice for downloading the various versions of NDK.
```
Comment (ryantrem) [inline README.md:112]:
```suggestion
- Android Studio is the tool of choice for downloading the various versions of NDK.
```

**Review by ryantrem — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (CedricGuillemet) [reply README.md:100]: You mean for apt-get ? they are system packages you only have to install once. And already installed packages won't be re-downloaded.

Comment (CedricGuillemet) [reply README.md:100]: For the path, you can add that to .bashrc or somewhere else depending on your distro/shell/tastes/...
https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path

**Review by Drigax — APPROVED** [review]

**Pull request merged**